internal/datacoord/compaction_trigger.go

```diff
@@ -108,7 +108,7 @@ func newCompactionTrigger(
 }
 
 func (t *compactionTrigger) start() {
-	t.globalTrigger = time.NewTicker(Params.DataCoordCfg.GlobalCompactionInterval.GetAsDuration(time.Second))
+	t.globalTrigger = time.NewTicker(Params.DataCoordCfg.MixCompactionTriggerInterval.GetAsDuration(time.Second))
```

Comment:
Is `GlobalCompactionInterval` deprecated? If so, please note it in the `doc` field.